content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md

```diff
@@ -14,21 +14,72 @@ CatalogContent:
 
 **Multi-factor Authentication (MFA)** is the use of more than one factor to prove the identity of a user.
 
+One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require users to create when initially signing up for an account.
+
 ## Factors
 
-One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require users to create when initially signing up for an account.
+There are, broadly speaking, five ways for a user to prove his identity. A user can prove his identity through
 
-Other factors, depending on the verification system, include the following:
+- Something the user knows
+- Something the user has
+- Something the user is
+- Some*where* the user is
+- Some*when* the user is
```

Comment:
```suggestion
- Something the user knows.
- Something the user has.
- Something the user is.
- Somewhere the user is.
- Somewhen the user is.
```

Reply:
I have taken away the italicizing of the text, but I'd rather not add punctuation mark to the list items for the sake of consistency given that we have not added punctuation to the other listed items.